.github/workflows/example.yml

```diff
@@ -41,14 +52,26 @@ jobs:
           insecure_skip_tls_verify: true
           namespace: ${{ env.TEST_NAMESPACE }}
 
+      - name: Create Image stream from the image
+        if: ${{ matrix.image-source == 'IMAGE_STREAM' }}
+        run: |
+          oc create is ${{ env.APP_NAME }}
+          oc import-image ${{ env.APP_NAME }} --from quay.io/redhat-github-actions/petclinic:latest
```

Comment:
should the image be `env.IMAGE_PATH` ?

Reply:
Yeah, it was in my mind but forgot to change it here :sweat_smile: